go/apps/ctrl/services/acme/user.go

```diff
@@ -105,7 +106,9 @@ func register(ctx context.Context, cfg UserConfig) (*lego.Client, error) {
 		return nil, fmt.Errorf("failed to encrypt private key: %w", err)
 	}
 
-	id, err := db.Query.InsertAcmeUser(ctx, cfg.DB.RW(), db.InsertAcmeUserParams{
+	id := uid.New("acme")
```

Comment:
it felt weird to have 2 tables with bigint ids and the rest with varchars
I think we should just keep it consistent